tests/test_constantq.py

```diff
@@ -184,6 +242,39 @@ def __test(note_min):
         yield __test, note_min
 
 
+def test_vqt_position():
+
+    # synthesize a two second sine wave at midi note 60
+
+    sr = 22050
+    freq = librosa.midi_to_hz(60)
+
+    y = np.sin(2 * np.pi * freq * np.linspace(0, 2.0, 2 * sr))
+
+    def __test(note_min):
+
+        C = np.abs(librosa.vqt(y, sr=sr, fmin=librosa.midi_to_hz(note_min)))**2
+
+        # Average over time
+        Cbar = np.median(C, axis=1)
+
+        # Find the peak
+        idx = np.argmax(Cbar)
+
+        assert idx == 60 - note_min
+
+        # Make sure that the max outside the peak is sufficiently small
+        Cscale = Cbar / Cbar[idx]
+        Cscale[idx] = np.nan
+        assert np.nanmax(Cscale) < 6e-1, Cscale
```

Comment:
This test is failing, presumably because of the reduced quality and higher redundancy at lower frequencies. The 60% (and likely 5% in the next assert) guarantee may be too strict for VQT. @bmcfee - any idea what do do here? Why were 0.6 and 0.05 chosen originally?

Reply:
IIRC those were chosen more or less by trial and error, and definitely not according to any rigorous theory.  Mainly the test there is to detect regressions (ie cqt quality getting worse somehow), not correctness.

Reply:
If I change it to 73% or higher, it looks like all cases would pass. However, I am not sure for the next assert, because I don't know if the unit test is aborted before it is run.

Should I change it to 73%?

Reply:
Changed to 73% and 23% in the following assert.